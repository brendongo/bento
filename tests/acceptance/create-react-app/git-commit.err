Running Bento checks...


âœ” 0 findings in  s
