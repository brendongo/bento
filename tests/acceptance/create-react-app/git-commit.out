[detached HEAD ] bad
 1 file changed, 1 insertion(+)
 create mode 100644 failure.py
